SwiftyJSONAPI/JSONAPIResource.swift

```diff
@@ -14,7 +14,9 @@ public enum JSONAPIResourceLoaded {
     case NotLoaded
 }
 
-typealias CachedResourceIds = Set<String>
+typealias ResourceType = String
+typealias ResourceId = String
+typealias CachedResourceIds = [ResourceType : ResourceId]
```

Comment:
This will not work, you need to have `[ResourceType: [ResourceId]]` since you have multiples ids per type